appcenter/ios/appcenter-rn.podspec

```diff
@@ -3,7 +3,7 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, '../package.json')))
 
 Pod::Spec.new do |s|
-  s.name              = package['name']
+  s.name              = 'appcenter-rn'
```

Comment:
Since that's the only that needs a suffix because no dash otherwise, how about `appcenter-core` ? That would make it more consistent with other modules (we also have Core terminology in the native SDK podspec).